apps/examples/wakaama_client/wakaama_client.c

```diff
@@ -394,6 +314,41 @@ int wakaamaclient_main(int argc, char *argv[])
 			printf("Read a resource value\n");
 			return 0;
 
+		case 'f': {
```

Comment:
Also most of 'case' returns 0 at the last, then when below code is needed??


 	dm_context.server_info.isBootstrap = bootstrap_requested;
 	prv_wakaama_start();
 	g_wakaama_initialized = 1;
        return 0;